common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -54,8 +56,15 @@ public void run() {
             // Do nothing.
         }
     };
+    private static final Runnable NOOP_TASK = new Runnable() {
```

Comment:
Honestly I would like to use some "dedicated" object. and as it is static it should not hurt anyway.